client/src/components/ReviewEntry.jsx

```diff
@@ -1,52 +1,64 @@
-import React, { useState, useEffect } from 'react';
+import React from 'react';
+import ReviewProfile from './ReviewProfile.jsx';
 
 const ReviewEntry = (props) => {
-  const { review } = props;
+  const { review, updateVote } = props;
+
+  let date = new Date(review.date_posted);
+  let options = { month: 'long', day: 'numeric' };
```

Comment:
I am very curious. What is happening here?

Reply:
the toLocaleString parses the date taking in a "locale" option, in this case en-US, and options, which converts the month to its 'long' version, which would be the string, and the 'numeric' day, so just the date!